common/cultures/dvg_cultures.txt

```diff
@@ -4078,106 +4078,107 @@ venetian={
 	}
 
 	male_common_first_names={
-	 #North Italian Names
-	 Agustin	
-	 Austin	
-	 Alvise	
-	 Ambroso	
-	 Andrea	
-	 Angelo	
-	 Annibale
-	 Antonio	
-	 Anzolo	
-	 Aurelio	
-	 Baldisse
-	 Bartholo
-	 Bastiano
-	 Bastian	
-	 Battista
-	  Batista
-	 Benedett
-	  Benetto
-	 Bernardi
-	 Biasio	
-	 Bortolo	
-	 Bortholo
-	 Carlo	
-	 Claudio 
-	 Cristofo
-	 Dionisio
-	 Domenico
-	 Filippo
-     Francesco
-     Galeazzo
-     Giovanni Battista
-     Giovanbattista
-     Giambattista
-     Giovanni
-     Giulio
-     Iulio
-     Guglielmo
-     Hieronimo
-     Iacomo
-     Giacomo
-     Jacomo
-     Jacopo
-     Ippolito
-     Iseppo
-     Lodovico
-     Lorenzo
-     Luca
-     Lundardo
-     Magno
-     Marcantonio
-     Marco
-     Marcello
-     Miro Nicolo
-     Paolo    
-     Pier     
-     Antonio  
-     Petro    
-     Rocco    
-     Salvador 
-     Stefano  
-     Teodor   
-     Tommaso
-     Valentin
-     Valerio
-     Ventura
-	 Paulo   
-     Vincenzo
-     Vittorio
-	 Zacharia
-	 Zago    
-	 Zordan  
-	 Zorzi
-	 Zuan
-	 Venturo     
-	 Zuanne      
-	 Antonio
-	 Zuandomenego
-	 Zuan
-	 Giacomo
-	 Zanetto
-	 #Unique Venice Names
-	 Silvestro
-	 Pietro
-	 Vitale
-	 Domenico
-	 Maurizio
-	 Orso
-	 Teodato
-	 Agnello
-	 Giustiniano
-	 Marino
-	 Albano
-	 Bartolomeo
-	 Gasparo
-	 Piero
-	 Pantaleone
-	 Pompeo
-	 Polo
-	 Jacobello
+	#North Italian Names
+	Agustin	
+	Austin	
+	Alvise	
+	Ambroso	
+	Andrea	
+	Angelo	
+	Annibale
+	Antonio	
+	Anzolo	
+	Aurelio	
+	Baldisse
+	Bartholo
+	Bastiano
+	Bastian	
+	Battista
+	Batista
+	Benedett
+	Benetto
+	Bernardi
+	Biasio	
+	Bortolo	
+	Bortholo
+	Carlo	
+	Claudio 
+	Cristofo
+	Dionisio
+	Domenico
+	Filippo
+    Francesco
```

Comment:
Same issue here as the last PR, can you make the spacing consistent with the rest of the file?

Reply:
I'm going to reupload the file, but it was fixed on my end when I uploaded it originally